### PR TITLE
Adding scrapped data to database and extracting it

### PR DESCRIPTION
**Pull request by crmgogo**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
-from flask import Flask, render_template, request, flash, redirect, url_for
-from .scraper import scrape_df
+from flask import Flask, render_template, request, flash
+# from .database import read_from_db
+from flask_pymongo import PyMongo
```
Comment (kartikrawool) [inline src/app.py:3]: Did you add this to requirements.txt?

**Review by TejasPrabhu — COMMENTED** [review]
Do we need the database.py file?

File: src/app.py (hunk 1 above)
Comment (crmgogo) [reply src/app.py:2]: Yes

**Comment by kartikrawool** [comment]
Use the extension mentioned in extensions.json to check flake8 on local machine

**Comment by TejasPrabhu** [comment]
closes #34

**Pull request merged**